src/main/python/ui/postbuilder.py

```diff
@@ -237,9 +240,12 @@ def retranslateUi(self, postbuilder):
         self.comboBox.setItemText(1, _translate("postbuilder", "TV Show"))
         self.spectrumLabel.setText(_translate("postbuilder", "Spectrum URL"))
         self.sourcePicker.setItemText(0, _translate("postbuilder", "Disc"))
-        self.sourcePicker.setItemText(1, _translate("postbuilder", "Amazon Prime"))
-        self.sourcePicker.setItemText(2, _translate("postbuilder", "iTunes"))
-        self.sourcePicker.setItemText(3, _translate("postbuilder", "Netflix"))
+        self.sourcePicker.setItemText(1, _translate("postbuilder", "Apple TV+"))
```

Comment:
I suggest moving these into code rather than setting them in the ui file (could potentially extend that later to be something a user can add to via preferences)

Reply:
Good idea, moved to code iterating over a list.  It'll be easier to maintain even if it's not user defined.